eng/build.yml

```diff
@@ -146,7 +146,7 @@ phases:
           packagesToPush: '$(Build.SourcesDirectory)/artifacts/packages/$(_BuildConfig)/NonShipping/VS.*.nupkg'
           nuGetFeedType: external
           publishFeedCredentials: 'DevDiv - VS package feed'
-        condition: and(succeeded(), eq(variables['_PushToVSFeed'], 'true'), eq(variables['_DotNetPublishToBlobFeed'], 'true'), or(eq(variables['_BuildArchitecture'], 'x64'), eq(variables['_BuildArchitecture'], 'x86')))
+        condition: and(succeeded(), ne(variables['PostBuildSign'], true), eq(variables['_PushToVSFeed'], 'true'), eq(variables['_DotNetPublishToBlobFeed'], 'true'), or(eq(variables['_BuildArchitecture'], 'x64'), eq(variables['_BuildArchitecture'], 'x86')))
```

Comment:
Should we include arm64 here?

Reply:
good question,

> The SDK repo had a bunch of files that were created using crossgen with an explicit certificate list. for signing, Not sure if the heuristics will select the right items during post build.

Post build signing doesn't use a heuristic, it just uses the certificate info from Signing.props. The latest SDKs should have the correct info, if you want to double check.

Reply:
These are only SDK files, correct?  From a VS insertion perspective, the only arm packages we need are the apphost packs.  I believe you found that I was getting the signed runtime files and only the sdk/installer files were unsigned.  If so, then maybe we're good here as apphost comes from runtime repo, correct?